packages/nimble-components/src/table/specs/table-column-width-hld.md

```diff
@@ -0,0 +1,157 @@
+# Table Column Widths
+
+## Problem Statement
+
+We need to provide users the means for changing the widths of individual columns as needed. This should be able to be accomplished both through interactive means as well as via a programmatic API.
+
+## Links To Relevant Work Items and Reference Material
+
+- [#873 Programmatically resize column width](https://github.com/ni/nimble/issues/873)
+- [#846 Interactively resize column width](https://github.com/ni/nimble/issues/846) 
+- [Table Design Doc](https://xd.adobe.com/view/5b476816-dad1-4671-b20a-efe796631c72-0e14/screen/d389dc1e-da4f-4a63-957b-f8b3cc9591b4/specs/)
+
+
+## Implementation / Design
```

Comment:
It might be worth having an "out-of-scope" section of the HLD. I think auto resizing of a column would go in that section and maybe also having different sizing modes for the table itself.